pages.es/common/chsh.md

```diff
@@ -0,0 +1,20 @@
+# chsh
+
+> Cambia el intérprete de comandos de inicio de sesión.
+> Más información: <https://manned.org/chsh>.
+
+- Cambia interactivamente el intérprete del usuario actual:
+
+`chsh`
+
+- Cambia el intérprete del usuario actual por otro en específico:
+
+`chsh -s {{ruta/al/intérprete}}`
+
+- Cambia el intérprete de otro usuario:
+
+`chsh -s {{ruta/al/intérprete}} {{nombre_de_usuario}}`
+
+- En[l]ista los intérpretes disponibles:
```

Comment:
En[l]ista > [l]ista

Reply:
What do you mean by `X` > `Y`, replace `X` for `Y`?

Reply:
As far as I know there should not be much trouble using the verb `enlistar`, [it is synonymous to `listar`](https://dle.rae.es/listar).

Maybe it could be easier for users to understand the `-l` option if the mnemonic is right at the start.